File: spring-boot/src/main/java/zencode/sb/portfolio/security/SecurityConstants.java
package zencode.sb.portfolio.security;

/**
 * @author Koert Zeilstra
 */
public class SecurityConstants {
  public static final String SECRET = "SecretKeyToGenJWTs";
  public static final long EXPIRATION_TIME = 60000; // 864_000_000; // 10 days
  public static final String TOKEN_PREFIX = "Bearer ";
  public static final String HEADER_STRING = "Authorization";
  public static final String SIGN_UP_URL = "/users/sign-up";
  public static final String HEALTH_URL = "/health/check";
}

File: stock-sb/src/main/kotlin/nl/zencode/port/stock/security/SecurityConstants.kt
package nl.zencode.port.stock.security

/**
 * @author Koert Zeilstra
 */
object SecurityConstants {
  const val SECRET = "SecretKeyToGenJWTs"
  const val EXPIRATION_TIME: Long = 864000000 // 10 days
  const val TOKEN_PREFIX = "Bearer "
  const val HEADER_STRING = "Authorization"
  const val SIGN_UP_URL = "/users/sign-up"
  const val HEALTH_URL = "/health"
}